samples/v1.2/Elements/Image.DataUri.json

```diff
@@ -0,0 +1,11 @@
+{
+    "type": "AdaptiveCard",
+    "version": "1.0",
+    "body": [
+        {
+            "type": "Image",
```

Comment:
>             [](start = 0, length = 12)

We're supposed to be using tabs here. You should probably take a moment to enable [EditorConfig support for Android Studio](https://stackoverflow.com/questions/44694132/editorconfig-in-android-studio). :)